.travis.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       python: 3.6
     - stage: precheck
       env: PURPOSE='Dependency Check'
-      script: ./scripts/dependency/check.sh
+      script: ./scripts/ci/dependency_check.sh
       python: 3.6
 #    - stage: verify
```

Comment:
Shouldn't there be more edits to the CI where we install the CLI from the requirements file, such as before running tests?

Reply:
Ah, there's a family of PRs coming as I update the tests and installers. This was just to get started before I unleash all of those, to make sure we are all on the same page in terms of general approach.

Reply:
So I think the existing script is more self-documenting in terms of what someone should do to fix the issue. I don't see where it calls out which requirements file is the offending one. Also, is this only being run for Python 3 Linux since that is what the CI runs?  What about the other requirements files? How will we know they need to be updated?

Reply:
Mind if I address this in a follow-up? I think the points raised here are important, but I think it's more critical that I get started on overhauling the installers to use these dependency lists.

Reply:
Track in #9795